README.md

```diff
@@ -27,6 +27,36 @@ Flask in development mode
 FLASK_ENV=development FLASK_APP=extension_explorer/views.py flask run
 ```
 
+# I18n
+
+## Compile messages:
+
+This needs to be done on each computer for i18n to work. If updating messages from transifex this has to be done last.
```

Comment:
Can this be clarified?

> This needs to be done on each computer for i18n to work

Does This mean the "compile messages" section or the whole "I18N" section?

> If updating messages from transifex this has to be done last.

Done lost compared to what? 


Is this whole I18N section basically optional? If your a dev just running the site you don't need to touch it, but 
  *  if you want to get updated translations 
  *  you change the messages that are in the templates

... then you need to run the whole section?

So it's either - don't change the messages and just totally ignore the I18N section - or - do change them and run the whole I18N section? Is that right? Can that be clarified if so?